reaktive/src/commonTest/kotlin/com/badoo/reaktive/observable/SkipTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.testutils.getOnNextValues
+import com.badoo.reaktive.testutils.test
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class SkipTest {
+    @Test
+    fun `should skip n values`() {
```

Comment:
Function names with spaces do not work for JS. Please check our test naming style.

Reply:
Fixed 👍